src/components/pages/ViewAllMeetings/VAMData.js

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from 'react';
+import MeetingInfo from './MeetingInfo';
+
+const VAMData = props => {
+  const { meetingData } = props;
+
+  return (
+    <div>
```

Comment:
i think you could get rid of this div and just put it in the <> </>  that way its easier to style, and it doesn't  show up in the dom with so many divs inside. 
Because right now the whole component is like 
```
<div>
    <div>
        <div>
         //meeting info
        </div>
    </div>

</div

```